ovs/dal/hybrids/albabackend.py

```diff
@@ -166,7 +166,7 @@ def _presets(self):
                 osds[node_id] = 0
                 for slot_id, slot_data in slots.iteritems():
                     for osd_id, osd_data in slot_data['osds'].iteritems():
-                        if osd_data['status'] in ['claimed', 'warning']:
+                        if osd_data['status'] in ['ok', 'warning']:
```

Comment:
if osd_data['status'] in [AlbaNode.OSD_STATUSES.OK, AlbaNode.OSD_STATUSES.WARNING]: use the enums pls